bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Display.java

```diff
@@ -1960,7 +1965,7 @@ public int getIconDepth () {
 	};
 }
 
-ImageList getImageList (int style, int width, int height) {
+ImageList getImageList (int style, int width, int height, int currentZoomLevel) {
```

Comment:
Rename the parameter to zoom as well for consistency. See all the other image List methods as well

Reply:
Done